pkg/git/git.go

```diff
@@ -161,7 +161,12 @@ func GetGitPlan(
 
 		var cloneArg string
 		if parsedGitPlan.Protocol == "https" {
-			cloneArg = "https://" + parsedGitPlan.Repository
+			cloneToken := os.Getenv("GIT_CLONE_TOKEN")
```

Comment:
We should give it a name this is unambiguous about what it is used for. I.e. SHUTTLE_GIT_TOKEN.

Reply:
Done.